lips/lip-6.mdx

```diff
@@ -0,0 +1,140 @@
+* [`rotate_authentication_key`](https://github.com/libra/libra/blob/master/language/stdlib/transaction_scripts/rotate_authentication_key.move) - to rotate the authentication key of the Validator Operator Account
+
+## Formal verification guarantees on the Move Implementation
+TODO: These may change as verification work proceeds.
+
+* The Move framework represents Validator Nodes and Validator Operators as addresses.  Each such address has a published account.
+* Every account has a unique role (see roles and permissions)
+* Each Validator account has a Validator role.
+* Each ValidatorOperator account has a ValidatorOperator role.
+* Only LibraRoot or a Validator Operator of a Validator Owner which is currently in the Validator Set can trigger reconfiguration
+* Every change to a Validator Set triggers a reconfiguration when LibraConfig::NewEpochEvent is emitted
+* Only a Validator Owner (role) can set its Validator Operator; for each Validator Owner account this permission is given only to the Validator Owner (itself).
+* Only a Validator Operator of a Validator Owner who is present in the Validator Set can update a Validator Config in the Validator Set
+* The size of the Validator Set can only change by at most one in a block
+* Multiple calls to add_validator or remove_validator in the same block are prohibited except in genesis
+* Both functions register_validator_config or set_validator_config_and_reconfigure check the validity of the consensus key and abort if an invalid key is given
+* A Validator Owner account can only appear once in a Validator Set
+* (currently) If the Validator's Operator is not in the Validator Set, the Validator Owner itself acts as its own Validator Operator
+* If the Validator Owner changes the Validator Operator, its old Validator Config does not change in the Validator Set until a new Validator Operator sets a new Validator Config and explicit calls an update (LibraSystem::update_config_and_reconfigure) or the Association calls add/remove (LibraSystem::add_validator/LibraSystem::remove_validator).
+* Disciplined aborts
+
+# Notes
+In the first version of the Libra Protocol, Validator Owners will not maintain ownership of their Libra accounts; rather the LibraRoot account will run Validator Owners' operations on their behalf. Validator Operators on the other hand will maintain full ownership of their Libra accounts. Gradually, the Validator Owners will start managing their accounts themselves. A Validator Owner will send their new account key to the Association, the Association will rotate their account key to this new key and right after this rotation transaction gets executed, the Validator Owner starts to maintain its Libra account.
```

Comment:
I'd prefer for us to release a new lip that updates our governance policy and strike this when appropriate rather than try to accommodate versioning now.